src/constants.ts

```diff
@@ -2,3 +2,5 @@ export const TARGET_MAX_RADIUS = 2000
 export const TARGET_MIN_RADIUS = 0
```

Comment:
This file shouldn't be global, it's an antipattern

Reply:
What do you mean? That I should split constants into their modules?